addons/payment_asiapay/models/payment_provider.py

```diff
@@ -14,6 +14,14 @@ class PaymentProvider(models.Model):
     code = fields.Selection(
         selection_add=[('asiapay', "AsiaPay")], ondelete={'asiapay': 'set default'}
     )
+    asiapay_brand = fields.Selection(
+        string="Brand",
```

Comment:
Sorry, I suggest [[1]](https://github.com/odoo/odoo/pull/108971#discussion_r1072218999) [[2]](https://github.com/odoo/odoo/pull/108971#discussion_r1072222669) a change that was meant to simplify the code, but it risks to add complexity later on. Please revert the changes I asked you to make and do as follows:
```suggestion
        string="AsiaPay Brand",
```
This avoids having multiple brand fields fighting for the "Brand" string on the `payment.provider` model. The field must then be renamed in the view back to "Brand".

Reply:
No worries, I updated the code again. Please take a look :)